Doc/library/zipimport.rst

```diff
@@ -188,17 +188,21 @@ Here is an example that imports a module from a ZIP archive - note that the
 
 .. code-block:: shell-session
 
-   $ unzip -l example.zip
-   Archive:  example.zip
+   $ unzip -l pip.pyz
+   Archive:  pip.pyz
      Length     Date   Time    Name
     --------    ----   ----    ----
-        8467  11-26-02 22:30   jwzthreading.py
+       32145  05-10-25 10:00   pip/__main__.py
     --------                   -------
-        8467                   1 file
-   $ ./python
-   Python 2.3 (#1, Aug 1 2003, 19:54:32)
+       32145                   1 file
+
+Now for the Python:
+
+.. code-block:: python
+
```

Comment:
This is overly complex, please revert. The point is to have a simple and quick example.